BDArmory/Distribution/GameData/BDArmory/Parts/towMissile/towMissile.cfg

```diff
@@ -84,7 +84,7 @@ PART
 	  maxStaticLaunchRange = 3750
 	  audioClipPath = BDArmory/Sounds/rocketLoop
 	  boostClipPath = BDArmory/Sounds/rocketLoop
-	  exhaustPrefabPath = BDArmory/Models/exhaust/smallExhaust
+	  //exhaustPrefabPath = BDArmory/Models/exhaust/smallExhaust
```

Comment:
Apparently the TOW missile FX is baked into the model, so this was adding the extra sideways exhaust plume